Source/Request.swift

```diff
@@ -816,17 +831,65 @@ public class Request {
     ///
     /// - Note: When waiting for a `Request`'s `URLRequest` to be created, only the last `handler` will be called.
     ///
-    /// - Parameter handler: Closure to be called when the cURL description is available.
+    /// - Parameters:
+    ///   - queue:   `DispatchQueue` on which `handler` will be called. `.main` by default.
+    ///   - handler: Closure to be called when the cURL description is available.
     ///
     /// - Returns:           The instance.
     @discardableResult
-    public func cURLDescription(calling handler: @escaping (String) -> Void) -> Self {
+    public func cURLDescription(on queue: DispatchQueue = .main, calling handler: @escaping (String) -> Void) -> Self {
```

Comment:
As a note, this is not truly an API compatible change, so needs to change.

Reply:
Added a separate override of `cURLDescription` to add this parameter while not breaking the previous version.